CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+Contributing to WildFly Elytron
+======================
+
+- [Contributing to WildFly Elytron](#contributing-to-wildfly-elytron)
+  * [Pre-requisites](#pre-requisites)
+  * [Issues](#issues)
+    + [Good First Issues](#good-first-issues)
+  * [Setup Development Environment](#setup-development-environment)
+  * [Coding Guidlines](#coding-guidlines)
+
+
+## Pre-requisites 
+To contribute, you will first need to fork the [wildfly-elytron](https://github.com/wildfly-security/wildfly-elytron) repository. 
+
+This can be done by looking in the top-right corner of the repository page and click "Fork".
+![fork](https://docs.github.com/assets/images/help/repository/fork_button.jpg)
+
+The next step is to clone your newly forked repository onto your local workspace. This can be done by going to your newly forked repository, which should be at `https://github.com/USERNAME/wildfly-elytron`. 
+
+Then there will be a green button that says "Code", select that then copy the url.
+
+![clone](https://i.imgur.com/N7KAxlg.png)
+
+Then in your terminal paste the following command
+`git clone [URL]`
+where you replace `[URL]` with the URL that you copied.
+
+Now you have the repository on your computer!
+
+## Issues
+The WildFly Elytron project uses JIRA to manage issues. All issues can be found here https://issues.redhat.com/projects/ELY/issues. 
+
+To create a new issue, comment on an existing issue, or assign an issue to yourself, you'll need to first [create a JIRA account here](https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/registrations?client_id=jbossorg&response_type=code&scope=openid&redirect_uri=https%3A%2F%2Fsso.jboss.org%2Flogin%3Fservice%3Dhttps%253A%252F%252Fissues.redhat.com%26ffp%3DRedHatExternalProvider).
+
+
+### Good First Issues
+Want to contribute to the WildFly Elytron project but aren't quite sure where to start? Check out our issues with the `good-first-issue` label. These are a triaged set of issues that are great for getting started on our project. [These can be found here](https://issues.redhat.com/issues/?filter=12364234). 
+
+Once you have selected an issue you'd like to work on, make sure it's not already assigned to someone else. Then, remember to assign it to yourself to prevent someone else from also working on the same issue.
+
+![jira](https://i.imgur.com/uEz7v32.png)
```

Comment:
Let's add this file locally if possible.

Reply:
Added